client/src/components/PropertiesList.tsx

```diff
@@ -77,17 +126,40 @@ export type PortfolioAnalyticsEvent = (
 ) => void;
 
 const PropertiesListWithoutI18n: React.FC<
-  withMachineInStateProps<"portfolioFound"> & withI18nProps
+  withMachineInStateProps<"portfolioFound"> &
+    withI18nProps & { addressPageRoutes: AddressPageRoutes; isVisible: boolean }
 > = (props) => {
-  const { i18n } = props;
-  const { width: windowWidth, height: windowHeight } = Helpers.useWindowSize();
+  const { i18n, isVisible } = props;
   const locale = (i18n.language as SupportedLocale) || defaultLocale;
   const useNewPortfolioMethod = props.state.context.useNewPortfolioMethod || false;
-  const portfolioFiltersEnabled = process.env.REACT_APP_PORTFOLIO_FILTERS_ENABLED === "1";
+  const portfolioFiltersEnabled = process.env.REACT_APP_PORTFOLIO_FILTERS_ENABLED === "1" || true;
 
   const addrs = props.state.context.portfolioData.assocAddrs;
   const rsunitslatestyear = props.state.context.portfolioData.searchAddr.rsunitslatestyear;
 
+  const { filterContext } = React.useContext(FilterContext);
+  const { viewType } = filterContext;
+
+  // avoid loading any components until portfolio tab is viewed
+  const [everVisible, setEverVisible] = React.useState(false);
+
+  React.useEffect(() => {
+    setEverVisible((prev) => prev || isVisible);
+  }, [isVisible]);
+
+  // only load the map or table when viewed, and avoid reloading when switching between
+  const [viewsEverVisible, setViewsEverVisible] = React.useState({
+    table: viewType === "table",
+    map: viewType === "map",
+  });
+
+  React.useEffect(() => {
+    setViewsEverVisible((prev) => ({
+      table: prev.table || viewType === "table",
+      map: prev.map || viewType === "map",
+    }));
+  }, [viewType]);
+
```

Comment:
added a comment in the code here - we don't expect it to be false again, but in future we might be able to land on the portfolio page in map or table view, so this will be helpful then.